src/main/java/pl/simpleascoding/tutoringplatform/service/user/UserServiceImpl.java

```diff
@@ -26,17 +26,18 @@
 @RequiredArgsConstructor
 class UserServiceImpl implements UserService {
 
-    private final UserRepository userRepository;
-    private final TokenRepository tokenRepository;
-    private final PasswordEncoder passwordEncoder;
-
-    private final JavaMailSender mailSender;
-
     private static final String REGISTRATION_MAIL_SUBJECT = "Confirm your email";
     private static final String REGISTRATION_MAIL_TEXT = "Hi %s, please visit the link below to confirm your email address and activate your account: \n%s";
     private static final String REGISTRATION_LINK = "%s/confirm-registration?tokenValue=%s";
-
     private static final String USER_NOT_FOUND_MSG = "User with \"%s\" username, has not been found";
+    private static final String PASSWORD_CHANGE_CONFIRMATION_MAIL_SUBJECT = "Confirm your password change";
+    private static final String PASSWORD_CHANGE_CONFIRMATION_MAIL_TEXT = "Hi %s, please visit the link below to confirm your password change: \n%s";
```

Comment:
**PASSWORD_CHANGE_CONFIRMATION_MAIL_TEXT**
in this particular case should be:
**PASSWORD_CHANGE_CONFIRMATION_MAIL_CONTENT**

Reply:
Done